File: src/main/java/juuxel/lakeside/Lakeside.java
package juuxel.lakeside;

import juuxel.lakeside.biome.LakesideBiomes;
import net.fabricmc.api.ModInitializer;
import net.minecraft.util.Identifier;

public final class Lakeside implements ModInitializer {
    public static final String ID = "lakeside";

    public static Identifier id(String path) {
        return new Identifier(ID, path);
    }

    @Override
    public void onInitialize() {
        LakesideBiomes.init();
    }
}

File: src/main/kotlin/juuxel/lakeside/Lakeside.kt
package juuxel.lakeside

import juuxel.lakeside.biome.LakesideBiomes
import net.fabricmc.api.ModInitializer
import net.minecraft.util.Identifier

object Lakeside : ModInitializer {
    fun id(name: String) =
        Identifier("lakeside", name)

    override fun onInitialize() {
        LakesideBiomes.init()
    }
}
